docs/reference/announcements/870.md

```diff
@@ -0,0 +1,136 @@
+---
+id: announcements-870
+title: "8.7 Announcements"
+description: "Important announcements including deprecation & removal notices"
+---
+
+Important changes and updates for the Camunda 8.7 release that customers should be aware of are summarized below.
+
+## What's coming in Camunda 8.7?
+
+| Scheduled release date | Scheduled end of maintenance | Release notes                                        | Blog                                                                                            |
+| :--------------------- | :--------------------------- | :--------------------------------------------------- | :---------------------------------------------------------------------------------------------- |
+| 11 February 2025       | 11 August 2026               | [8.7 release notes](/reference/release-notes/870.md) | [Announcing Camunda 8.7](https://camunda.com/blog/2024/11/camunda-8-7-releasing-february-2025/) |
+
+Camunda 8.7 is scheduled for release on 11 February, 2025.
+
+- [API updates](#api-updates-saasself-managed)
+- [Identity management updates](#identity-management-updates-saasself-managed)
+- [Installation and deployment updates](#installation-and-deployment-updates-self-managed)
+- [Camunda Java client and Camunda Spring SDK](#camunda-java-client-and-camunda-spring-sdk-self-managed)
+
+:::info
+
+The following blog posts also provide more information on the changes included in the Camunda 8.7 release:
+
+- Upcoming API Changes in Camunda 8
+- Introducing Enhanced Identity Management in Camunda 8.7
+- Streamlined Deployment with 8.7
+
+:::
+
+## API updates <span class="badge badge--long" title="This feature affects SaaS">SaaS</span><span class="badge badge--long" title="This feature affects Self-Managed">Self-Managed</span>
+
+The 8.7 release includes API updates to support the move to a [Camunda 8 REST API](/apis-tools/camunda-api-rest/camunda-api-rest-overview.md) unified experience.
+
+### Camunda 8 REST API updates
+
+- New Query endpoints (with advanced search filtering) will be added for process entities (processes, decisions, user tasks, and forms). These will replace the component APIs (Tasklist, Operate) going forward.
+- New endpoints will allow you to manage and query users and resource permissions in an orchestration cluster.
+- All the Camunda 8 REST API endpoints will support resource-based authorizations to enable fine-grained permissions.
+- API terminology is aligned so technical assets have an identical, easily-understood, descriptive property name.
+
+### Deprecated: Operate and Tasklist REST APIs
```

Comment:
Could we add 'v1' API qualifier as it reads that we deprecate all Operate, Tasklist API's